package.json

```diff
@@ -27,8 +27,9 @@
   "license": "MIT",
   "dependencies": {
     "@babel/runtime": "^7.12.5",
-    "cosmiconfig": "^7.0.0",
-    "resolve": "^1.19.0"
+    "lilconfig": "^2.0.4",
+    "resolve": "^1.19.0",
+    "yaml": "^1.10.2"
```

Comment:
there is just a recent security vulnerability on yaml <2.2.2, is it possible to take an upgraded version? Thanks
https://security.snyk.io/vuln/SNYK-JS-YAML-5458867